client/state/selectors/is-any-plugin-active.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Returns a Boolean indicating if a site has an active plugin present
+ * from a given array. This is useful for jetpack connected sites.
+ *
+ * @param {Object} state - Global state tree
+ * @param {Number} siteId - Site ID
+ * @param {Array} pluginSlugs - Array of plugin slugs for which to check
+ * @return {Boolean} - truthiness of a site having at least one of the given plugins active
+ */
+export default function isAnyPluginActive( state, siteId, pluginSlugs ) {
+	const sitePlugins = get( state, `plugins.installed.plugins[ ${ siteId } ]`, [] );
+	const matches = sitePlugins.filter(
```

Comment:
We could reuse the existing `getPlugins` [selector](https://github.com/Automattic/wp-calypso/blob/master/client/state/plugins/installed/selectors.js#L62) to simplify this (at which point we might not need a separate selector). For example:

```javascript
const activePlugins = getPlugins( state, siteId, 'active' );

return !! intersectionWith( activePlugins, blacklistPlugins, ( ap, bp ) => ap.slug === bp );
```

Reply:
Ooooh, I like this way better; added d842546 for evaluation.

Reply:
Thanks @kwight! 

As discussed on Slack, I don't have any strong objection about this approach. But I see some benefits on having the logic encapsulated in terms of code maintenance. Otherwise we should duplicate the same `getPlugins`/`intersectionWith` logic in `isWpAdminGutenbergEnabled`, `isGutenframeEnabled` and `getGutenbergEditorUrl`.

But I see your point of making that selector more abstract and less specific by keeping the array of slugs out of it.

On 5a8919e441e29daf2022d9e0be7c4a1813f94b28 I brought back your original more-generic `isPluginActive` selector and and moved the definition of the denied plugins list to `lib/plugins`.